examples/csv_postcards/create_postcards.py

```diff
@@ -0,0 +1,50 @@
+import lob
+import csv
+import sys
+
+lob.api_key = 'test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc'
+
+skipFirstLine  = True
+
+try:
+  sys.argv[1]
+except IndexError:
+  print "Please provide an input CSV file as an argument."
+  sys.exit()
+
+with open('postcard_front.html', 'r') as frontHtmlFile:
+    frontHtml = frontHtmlFile.read()
```

Comment:
above you indented 2 spaces, but here it's 4 spaces. shouldn't that cause errors in python?